### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by valzugg**
You can test this by for example using a broken link.
Currently if question fetching fails, choice fetching will show as if not failed, but this isnt necessarily a problem.

Capture

Should still do error handling for posting separately

### CODE REVIEW

File: src/components/ErrorPage.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Grid from '@material-ui/core/Grid';
+
+const style = {
+    body: {
+        backgroundColor: '#0078CC',
+        height: '100vh',
+    },
+    thankYouDiv: {
```
Comment (andersnylund) [inline src/components/ErrorPage.jsx]: This came probably from ThankyouPage?

File: src/translations/fi.json [hunk 2]
```diff
@@ -3,5 +3,6 @@
     "submitbutton": "Lähetä",
     "skipbutton": "Seuraavaan",
     "questionpage.required": "Tämä kysymys on pakollinen!",
-    "textfield.placeholder": "Vastauksesi"
+    "textfield.placeholder": "Vastauksesi",
+    "error.message": "Virhe"
```
Comment (andersnylund) [inline src/translations/fi.json]: Would 'Jotakin meni pieleen' be better?

File: src/translations/sv.json [hunk 3]
```diff
@@ -3,5 +3,6 @@
     "submitbutton": "Skicka",
     "skipbutton": "Skip",
     "questionpage.required": "Denna fråga är obligatorisk!",
-    "textfield.placeholder": "Ditt svar"
+    "textfield.placeholder": "Ditt svar",
+    "error.message": "Fel"
```
Comment (andersnylund) [inline src/translations/sv.json]: 'Någonting gick fel'

**Review by andersnylund — CHANGES_REQUESTED** [review]

File: src/translations/sv.json (hunk 3 above)
Comment (VonLatvala) [reply src/translations/sv.json]: Or, ’homma kusi 🙉’ maybe?

File: src/translations/fi.json (hunk 2 above)
Comment (VonLatvala) [reply src/translations/fi.json]: Or, ’homma kusi 🙉’, maybe?

File: src/translations/sv.json (hunk 3 above)
Comment (VonLatvala) [reply src/translations/sv.json]: Nåt gick snätt?

**Review by andersnylund — APPROVED** [review]

**Pull request merged**